### DISCUSSION TITLE
Jump to a specific Group Header

### DISCUSSION THREAD

**Discussion by pradeep-kumar-rs** [Dimibe/grouped_list#128] [Q&A] [answered]
Hi , How to Jump to a specific Group Header?

**Comment by juancabeco** [comment]
Hi, any answer?

**Reply by pradeep-kumar-rs** [reply]
the only solution i got is to use [sticky_grouped_list package](https://github.com/Dimibe/sticky_grouped_list)

**Comment by Dimibe** [comment, marked as answer]
Hi this request seems to be the same as in #116. Have a look at my answer there.
